### ISSUE TITLE
lastOriginUpdate Field Missing

### ISSUE THREAD

**Issue by Sidd27** [amodm/api-covid19-in#20]
lastOriginUpdate Field Missing is missing in below api
https://api.rootnet.in/covid19-in/unofficial/covid19india.org/statewise/history

And 

lastRefreshed and lastOriginUpdate have same value in below API all the time
https://api.rootnet.in/covid19-in/unofficial/covid19india.org

**Comment by amodm** [comment]
For covid19india.org data, `lastOriginUpdate` holds little relevance because that sheet is continuously getting updated, and they rely on a manual update of a particular field, which is easy to miss. So for them, I treat `lastOriginUpdate` as effectively meaningless. 

The raw patientData API was built first, so I retained those fields when building it, even though they were meaningless. Because it had already been put out, I chose not to delete them, lest someone's consumption break.

The statewise history API was built later, so learning from my previous experience with covid19india.org's data, I chose to ignore those fields for this API because they hold little value in their case. `lastRefreshed` is still there, but at the `source` level.
